packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1241,19 +1331,19 @@ class Auth {
     /**
      * A wrapper method for commerce-sdk-isomorphic helper: getPasswordLessAccessToken.
      */
-    async getPasswordLessAccessToken(parameters: LoginPasswordlessParams) {
-        const pwdlessLoginToken = parameters.pwdlessLoginToken
+    async getPasswordLessAccessToken(parameters: GetPasswordLessAccessTokenPublicParams) {
```

Comment:
Nit: 

I'd like to suggest not capitalizing the 'L' in Passwordless but we've capitalized it in isomorphic and everywhere else 

```suggestion
    async getPasswordLessAccessToken(parameters: GetPasswordlessAccessTokenPublicParams) {
```

Reply:
I see “Passwordless” with lowercase l is more correct. But for now I think is best to keep `getPasswordLessAccessToken` to stay consistent with `commerce-sdk-isomorphic@4.0.0` already shipped to NPM, since both packages are used directly by developers and having different names would be confusing. I think we could look at a coordinated fix across both packages in the future. What do you think?

Reply:
Agree, it would have to be a commerce-sdk-isomorphic v5 change since the package has already shipped